### ISSUE TITLE
include `scope` attribute for table headers

### ISSUE THREAD

**Issue by ericscheid** [calculuschild/marked-extended-tables#492]
This is for WCAG support.

https://www.w3.org/WAI/tutorials/tables/two-headers/

Essentially, ensure any rendered `<th>` elements incude a `scope="row"` or `scope="column"` attributes to indicate in which direction the `th` element is applied.

(NB. currently all `th` elements from marked-tables are `column`, ie. in the first or leading rows. This might change.)

**Comment by ericscheid** [comment]
See also #493.

**Comment by dbolack-ab** [comment]
I'll take this one @calculuschild
